dev/reference/tools/error-case-handling/lightning-misuse-handling.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Common misuses of TiDB Lightning
+category: reference
+---
+
+# Common misuses of TiDB Lightning
+
+This article introduces common error scenarios in using [TiDB Lightning](/dev/reference/tools/tidb-lightning/overview.md) and corresponding solutions.
+
+## Error: `checksum mismatched remote vs local`
+
+The following error was encountered during data import:
+
+```log
+Error: checksum mismatched remote vs local => (checksum: 3828723015727756136 vs 7895534721177712659) (total_kvs: 1221416844 vs 1501500000) (total_bytes:237660308510 vs 292158203078)
+```
+
+### Reasons
+
+* TiDB Lightning has been used to import data previously. This means the data was not cleaned in the corresponding [checkpoint](/dev/reference/tools/tidb-lightning/checkpoints.md). You can check the first launch log in TiDB Lightning to confirm:
+
+    * `[checkpoint] driver = file`. If the log corresponding to TiDB Lightning data import shows `open checkpoint file failed, going to create a new one`, then `checkpoint` is cleaned properly. Otherwise, the remaining data may lead to imported data missing.
```

Comment:
```suggestion
    * When `[checkpoint] driver = file` and the log corresponding to TiDB Lightning data import's start time shows `open checkpoint file failed, going to create a new one`, then the `checkpoint` is cleaned correctly. Otherwise, the remaining data may lead to imported data missing.
```